library/alloc/src/string.rs

```diff
@@ -2224,6 +2224,47 @@ impl ToString for char {
     }
 }
 
+#[stable(feature = "u8_to_string_specialization", since = "1.999.0")]
```

Comment:
What is with the since tag?

Reply:
Already fixed on master.

Reply:
Ah sorry, wasn't sure what to put for that. Yes 1.54 makes sense, but at the time I didn't want to presume so put something _a little_ further out.